documentation/docs/reference/configuration.md

```diff
@@ -0,0 +1,101 @@
+---
+description: This section describes the configuration parameters and their types for INX-POI.
+keywords:
+- IOTA Node 
+- Hornet Node
+- POI
+- Proof-Of-Inclusion
+- Proof
+- Inclusion
+- Configuration
+- JSON
+- Customize
+- Config
+- reference
+---
+
+# Core Configuration
+
+INX-POI uses [JSON](https://www.json.org) for configuration files.
+
+You can specify which config file INX-POI will use. Follow the `inx-poi` command with the `--config` or `-c` parameter and a path to file:
+
+```bash
+inx-poi -c config_defaults.json
+```
+
+To get the description of configuration parameters for the installed version of `inx-poi`, you can run the `inx-poi` command with `-h` and `--full` parameters:
+
+```bash
+inx-poi -h --full
+```
+
+## <a id="app"></a> 1. Application
```

Comment:
The custom ids don't work with Docusarus.   I'm guessing they are there for the raw GitHub content, but they don't work either.  

I don't understand the numbers. They don't seem to add much to the content and would make direct # links a bit dirtier.